apps/web/src/components/OccupationSelector.tsx

```diff
@@ -18,20 +19,20 @@ export const OccupationSelector = ({}) => {
             if (e.target.checked) {
               setFieldValue(
                 'occupation',
-                occupations.map((e: any) => e.id),
+                data.map((e: any) => e.id),
               );
             } else {
               setFieldValue('occupation', []);
             }
           }}
-          checked={values.occupation.length === occupations.length}
+          checked={values.occupation.length === data.length}
```

Comment:
Ahhh, this will check Select all based on the length of the filtered result. I don't think that's the expected or desired result